### PR TITLE
Improve date consistency across FC

### PR DESCRIPTION
**Pull request by NakedFool**
## What does this PR do and why?

Fixes - https://github.com/jbx-protocol/juice-interface/issues/1926

The intention of this PR is to improve date consistency across the funding cycle section

I've just got one question for you @tomquirk 
image

What is the expected situation here since in `start` and `end` we already present the date in UTC format, should we on hover also present the same format or vice versa? I understand suffix addition but couldn't understand hover addition 🍻 

## Screenshots or screen recordings

https://user-images.githubusercontent.com/18723426/189296437-b5e4c45c-ec15-43fe-9a3c-7b5abd5a6b20.mov

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by tomquirk** [comment]
@NakedFool I think `start`/`end` are displayed in the user's local time. In that screenshot I posted, 8:12am is my local time (i'm UTC +9:30). 

I think we have two options:
1. 2022-09-05 8:12am +9:30
2. 2022-09-04 10:42pm UTC

I like the local time. But for consistency, perhaps we should go with the second option and display UTC time.

What do you think?

**Comment by NakedFool** [comment]
> @NakedFool I think `start`/`end` are displayed in the user's local time. In that screenshot I posted, 8:12am is my local time (i'm UTC +9:30).
> 
> I think we have two options:
> 
> 1. 2022-09-05 8:12am +9:30
> 2. 2022-09-04 10:42pm UTC
> 
> I like the local time. But for consistency, perhaps we should go with the second option and display UTC time.
> 
> What do you think?

Ooh, you are right, my bad.

I'd go with option 2, however, we should only show the UTC date on hover in `start` and `end` right?

I'm attaching a recording of the current workaround just so you don't have to pull the latest changes, lmk what you think 

https://user-images.githubusercontent.com/18723426/189839182-d13bd08c-b276-4f1a-a2f3-b7da962d6a31.mov

**Comment by NakedFool** [comment]
@tomquirk the only issue i'm seeing now is UI based, as you can see on the screenshot there isn't enough space in `start` so `UTC` goes in the next row. wdyt?  

image

**Comment by tomquirk** [comment]
@NakedFool I'm pushing a merge commit! 🙏

File: src/components/v1/shared/FundingCycle/FundingCycleDetails.tsx [hunk 1]
```diff
@@ -156,13 +156,23 @@ export default function FundingCycleDetails({
 
         {fundingCycle.duration.gt(0) && (
           <Descriptions.Item label={<Trans>Start</Trans>}>
-            {formattedStartTime}
+            <Tooltip title={formatDateToUTC(fundingCycle.start.mul(1000))}>
```
Comment (tomquirk) [inline src/components/v1/shared/FundingCycle/FundingCycleDetails.tsx:159]: **suggestion (non-blocking)**: maybe we can create a `secondsToMilliseconds` utility for all these in a follow-up.

File: src/utils/format/formatDate.tsx [hunk 2]
```diff
@@ -10,9 +10,14 @@ export const formatDate = (
   format = 'YYYY-MM-DD h:mma',
 ) => moment(BigNumber.from(dateMillis).toNumber()).format(format)
 
+export const formatDateToUTC = (
```
Comment (tomquirk) [inline src/utils/format/formatDate.tsx:13]: **praise** nice one!

File: src/utils/format/formatDate.tsx [hunk 3]
```diff
@@ -10,9 +10,14 @@ export const formatDate = (
...
+  dateMillis: BigNumberish,
```
Comment (tomquirk) [inline src/utils/format/formatDate.tsx:14]: **praise** thanks for including the unit in the variable name.

**Review by tomquirk — APPROVED** [review]
@NakedFool thanks so much for this, all looks great. Merging!

File: src/components/v1/shared/FundingCycle/FundingCycleDetails.tsx [hunk 4]
```diff
@@ -1,14 +1,14 @@
 import { parseEther } from '@ethersproject/units'
 import { Trans } from '@lingui/macro'
-import { Descriptions } from 'antd'
+import { Descriptions, Tooltip } from 'antd'
```
Comment (tomquirk) [inline src/components/v1/shared/FundingCycle/FundingCycleDetails.tsx:3]: **praise** thanks also for making the changes to V1!!!

**Pull request merged**